### ISSUE TITLE
Stop counting removed releases from pypi.org

### ISSUE THREAD

**Issue by soslan** [librariesio/libraries.io#2260]
Seems like libraries.io is wrongly counting removed releases. [Here](https://github.com/librariesio/libraries.io/blob/68b767c9821a2419ace330b00e8a92399c26dda5/app/models/project_score_calculator.rb#L134), for example. I think, whether a package follows SemVer or not, should not depend on removed releases, because developer can accidentally upload wrong file and then remove it.
Compare release history of passgen package on [pypy.org](https://pypi.org/project/passgen/#history) and [libraries.io](https://libraries.io/pypi/passgen/versions)

**Comment by katzj** [comment]
Seems reasonable to exclude releases that are marked as removed. Versions don't currently have the idea of being removed but that seems reasonable to add and then we could exclude when something has been removed.

**Comment by bretttolbert** [comment]
Yes, it's saying [my project](https://libraries.io/pypi/verbecc/sourcerank) doesn't follow SemVar even though I removed the one release that didn't from pypi.

**Comment by agronholm** [comment]
It's also too bad that making any prereleases messes up SemVer compatibility because PEP 440 and SemVer are mutually exclusive.
